packages/cli/src/commands/config/schema.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Command } from 'commander';
+import { stringify as stringifyYaml } from 'yaml';
+import { loadCliConfig } from '../../lib/config';
+
+export default async (cmd: Command) => {
+  const { schema } = await loadCliConfig({
+    args: [],
+    fromPackage: cmd.package,
+    mockEnv: true,
+  });
+
+  const data = schema.serialize();
```

Comment:
I realized that this isn't quite gonna be what we want here. It's not a real JSON Schema but just our serialization format that that has the split schema to be able to provide better error messages.

The schema merging process is here: https://github.com/backstage/backstage/blob/f8481d2357ac65447e5c7fcbbcd9cbf15867d818/packages/config-loader/src/lib/schema/compile.ts#L80

I think we can basically take the result of that and output instead, perhaps also setting some JSON Schema meta schema top-level metadata. Tbh I think we could just duplicate the code for now, but perhaps slap a TODO to bring it in from config-loader if it starts to deviate

Reply:
I opted to refactor out the merge part instead of copying. I hate copying code, almost more than sloppy code :P 
Not done any test yet.. not exactly sure how to go about it, and seeing there is almost no logic to cover any way...